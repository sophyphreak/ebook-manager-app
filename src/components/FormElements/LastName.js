import React, { Component } from 'react';

const LastName = (props) => (
  <label>
    Author Last Name:*
    <input type="text" value={props.value} onChange={props.onChange} />
  </label>
);

export default LastName;