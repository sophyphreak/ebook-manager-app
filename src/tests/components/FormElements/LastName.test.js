import React from 'react';
import { shallow } from 'enzyme';
import LastName from '../../../components/FormElements/LastName';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<LastName />);
});

test('should render LastName correctly', () => {
  expect(wrapper).toMatchSnapshot();
});