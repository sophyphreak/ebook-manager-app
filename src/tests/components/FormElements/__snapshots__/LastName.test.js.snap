// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LastName correctly 1`] = `
<BasicInput
  label="Author Last Name:*"
/>
`;
